examples/eth_provider/mix.exs

```diff
@@ -0,0 +1,27 @@
+defmodule EthProvider.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :eth_provider,
+      version: "0.1.0",
+      elixir: "~> 1.15",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger]
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      {:wrapeth, path: "../..", override: true}
```

Comment:
This is fine, but did you test this with a git URL instead of a path?

Reply:
I tested it with a Git URL initially but noticed that the Ecto repository does it this way. I thought that using the path is more robust in case we ever decide to remove the repository.😁